docs/docs/how_to/how-to-oracles.md

```diff
@@ -267,6 +267,19 @@ app.use(cors())
 
 :::
 
+## Foreign Calls
+
+When making foreign calls, you need to ensure that:
+
+1. The function is marked with the `unconstrained` keyword
+2. The function is properly defined in your backend implementation
```

Comment:
In the context of Noir, backends don't know about foreign calls as they're an unconstrained concept.